extensions/cpp/package.json

```diff
@@ -34,8 +34,8 @@
 					".c++",
 					".hpp",
 					".hh",
-					".hxx",
-					".h++",
+          ".hxx",
+          ".h++",
 					".h",
```

Comment:
To confirm, this is in 1.50, right. Well, one less thing to put in for C++ environs.

thanks